releasing/plugin-commands-publishing/src/publish.ts

```diff
@@ -233,8 +258,18 @@ Do you want to continue?`,
     packDestination,
   })
   await copyNpmrc({ dir, workspaceDir: opts.workspaceDir, packDestination })
+
+  const env = Object.assign({})
```

Comment:
How is this different from just:

```suggestion
  const env = {}
```